src/system/time.953

```diff
@@ -228,24 +228,24 @@ IFDEF DSTEU,[IFN DSTEU,[
 	;; else it's obviously DST
 ]]
 .ELSE [
-	CAML A,[<APR 1>*SPD+7200.]	;IF BEFORE 2AM APR 1,
-	CAML A,[<OCT 31.>*SPD+3600.]	;OR IF AFTER 1AM STANDARD TIME OCT 31,
+	CAML A,[<MAR <1+6>>*SPD+7200.]	;IF BEFORE 2AM MAR 7,
+	CAML A,[<NOV 1>*SPD+3600.]	;OR IF AFTER 1AM STANDARD TIME NOV 1,
 	 POPJ P,			;THEN OBVIOUSLY STANDARD TIME IS IN EFFECT
-	CAML A,[<APR 7>*SPD+7200.]	;IF BEFORE 2AM APR 7 STANDARD TIME,
-	CAML A,[<OCT 25.>*SPD+3600.]	;OR IF AFTER 1AM STANDARD TIME OCTOBER 25,
+	CAML A,[<MAR <7+6>>*SPD+7200.]	;IF BEFORE 2AM MAR 13 STANDARD TIME,
+	CAML A,[<NOV <1+6>>*SPD+3600.]	;OR IF AFTER 1AM STANDARD TIME NOV 7,
 	 JRST GLPDT3			;THEN NOT OBVIOUS
 ]
 CRDDST:	TLO E,100000	;DAYLIGHT SAVINGS TIME, SET BIT IN E
 	ADDI A,3600.	;CONTINUE TO LOCALIZE THE TIME THAT WILL BE RETURNED
 	POPJ P,
 
-		;IT IS NOW SOMETIME DURING THE WEEKS IN APR AND OCTOBER
+		;IT IS NOW SOMETIME DURING THE WEEKS IN MAR AND NOV
 		;WHEN IT MAY BE EITHER STANDARD OR DAYLIGHT SAVINGS TIME
 
 GLPDT3:	PUSH P,A	;SAVE # SECS
 	CAMG A,[JUL*SPD]	;IF NOT YET JULY,
-	 TDZA D,D	;THEN IT MUST BE APR, SET INDEX
-	  MOVEI D,1	;OCTOBER, SET INDEX
+	 TDZA D,D	;THEN IT MUST BE MAR, SET INDEX
```

Comment:
The comments only reflect the non-EU case.

Reply:
And the original comments are a bit misleading. If it's not yet July, it's the first half of the year (spring on the Northern hemisphere), otherwise it's the second half (fall). This is what is tested, to check which index in CRDSB to use.